pkgs/by-name/fr/freeshow/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  pkgs,
+  appimageTools,
+}:
+
+let
+  pname = "freeshow";
+  version = "1.2.1";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.dmg";
+          hash = "sha256-OO0uQ6oS1GKiBGz3Qt9jDCY+qdWlgTOY+SiLJB5xQ3c=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.AppImage";
+          hash = "sha256-RBI8IkxY6Xd36vCVDJy9sqpEisB/48hwQo+mg5XTCOs=";
+        };
+      }
+      .${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  appimageContents = appimageTools.extract { inherit pname version src; };
+
+  meta = {
+    homepage = "https://github.com/ChurchApps/FreeShow";
+    description = "Free and open-source, user-friendly presenter software";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ ByteSudoer ];
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
```

```suggestion
    sourceProvenance = [ lib.sourceTypes.binaryNativeCode ];
```
Avoid `with` scoping as much as possible. The maintainer list is fine, as more maintainers may be added in the future. (It's still being discussed if it's fine to keep `with lib.maintainers`. For now, keep it.)